diagnostic.md

```diff
@@ -71,13 +72,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative, because we are in 'project' and moving into 'cd planning' relative to where we are
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+I believe we could refer to it with a relative path, if the project name is unique and the photo name is unique.
```

Comment:
It is relative, but just to clarify,  the image file lives inside our current working directory, `project` ; as a result, an absolute path is not needed to access it.